packages/Telephony/Actions/BatchFixedLengthInput.cs

```diff
@@ -22,6 +22,7 @@ public class BatchFixedLengthInput : RegexAggregatorInput
         public const string Kind = "Microsoft.Telephony.BatchFixedLengthInput";
 
         private const string _dtmfCharacterRegex = @"^[\d#\*]+$";
+        private const string _interruptionMaskRegex = @"^[\d]+$";
```

Comment:
Minor: add comment like "digits only regex" for easier reading of the code ;)

Reply:
smart.